release-notes/releases-index.json

```diff
@@ -44,6 +47,7 @@
             "latest-runtime": "3.1.26",
             "latest-sdk": "3.1.420",
             "product": ".NET Core",
+            "release-type" : "lts",
```

Comment:
Just noticed this. The index has 3.1 as LTS and LTS, but the 3.1 releases.json has LTS and maintenance

Reply:
Thanks. Resolved.